olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -1,5 +1,5 @@
 schema: olm.channel
-name: stable
+channel: stable
```

Comment:
Just for my understanding. Why is this needed? Does "name" refer to something more general than a channel?

Reply:
The OLM docs are using both variants, `name: ` & `channel: ` quiet randomly. :) But it might not be a good sign that operator tests failed already.

Reply:
IMO `name: ` makes more sense here.

Reply:
I followed the https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#channel-promotion, In the example they use the channel key, Not sure about the schemas definitions difference.
Is it breaking something?

Reply:
Okay switching back to **name:**